doc/topics/releases/3006.rst

```diff
@@ -0,0 +1,137 @@
++--------------+---------------------+---------------------+
+| Debian 10    |         yes         |        yes          |
++--------------+---------------------+---------------------+
+| Debian 11    |         yes         |        yes          |
++--------------+---------------------+---------------------+
+| Raspbian 9   |         no          |        no           |
++--------------+---------------------+---------------------+
+| Raspbian 10  |         no          |        no           |
++--------------+---------------------+---------------------+
+| Raspbian 11  |         no          |        yes          |
++--------------+---------------------+---------------------+
+| Fedora 35    |         yes         |        yes          |
++--------------+---------------------+---------------------+
+| Fedora 36    |         yes         |        yes          |
++--------------+---------------------+---------------------+
+| MacOS        |         yes         |        yes          |
++--------------+---------------------+---------------------+
+| Windows      |         yes         |        yes          |
++--------------+---------------------+---------------------+
+
+
+Repo paths
+----------
+
++----------+-----------------------------------------------+-----------------------------------------+
+|     OS   | Tiamat Path                                   | Non-Tiamat Path                         |
++==========+===============================================+=========================================+
+| RHEL     | https://repo.saltproject.io/salt/py3/redhat/  | https://repo.saltproject.io/py3/redhat/ |
++----------+-----------------------------------------------+-----------------------------------------+
+| Ubuntu   | https://repo.saltproject.io/salt/py3/ubuntu/  | https://repo.saltproject.io/py3/ubuntu/ |
++----------+-----------------------------------------------+-----------------------------------------+
+| Debian   | https://repo.saltproject.io/salt/py3/debian/  | https://repo.saltproject.io/py3/debian/ |
++----------+-----------------------------------------------+-----------------------------------------+
+| Raspbian | https://repo.saltproject.io/salt/py3/debian/  | https://repo.saltproject.io/py3/debian/ |
++----------+-----------------------------------------------+-----------------------------------------+
+| Fedora   | Hosted on Fedora Repos                        | Hosted on Fedora Repos                  |
++----------+-----------------------------------------------+-----------------------------------------+
+| MacOS    | https://repo.saltproject.io/salt/py3/osx/     | https://repo.saltproject.io/osx/        |
++----------+-----------------------------------------------+-----------------------------------------+
+| Windows  | https://repo.saltproject.io/salt/py3/windows/ |  https://repo.saltproject.io/windows/   |
++----------+-----------------------------------------------+-----------------------------------------+
```

Comment:
**Question:** I have the same question as above re: linking to latest, minor, major for Tiamat.